packages/core/src/Provider/DRepProvider/types.ts

```diff
@@ -0,0 +1,22 @@
+import { Cardano, Provider } from '../..';
+
+export interface GetDRepInfoArgs {
+  id: Cardano.DRepID;
+}
+
+export interface GetDRepsInfoArgs {
+  ids: Cardano.DRepID[];
+}
+
+export interface DRepInfo {
+  id: Cardano.DRepID;
+  amount: Cardano.Lovelace;
+  active: boolean;
+  activeEpoch?: Cardano.EpochNo;
+  hasScript: boolean;
+}
+
+export interface DRepProvider extends Provider {
+  getDRepInfo: (args: GetDRepInfoArgs) => Promise<DRepInfo>;
```

Comment:
```suggestion
```
Precedent it to just have a single endpoint for other providers, with the consumer sending an array of one item to perform individual lookups. e.g https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/core/src/Provider/ChainHistoryProvider/types.ts

Reply:
Ah good point! Precedent is then, in-fact, not clear! OK as-is